lib/src/excel.dart

```diff
@@ -48,22 +71,42 @@ class ExcelDart {
   /// Waits for Office APIs to be ready and then creates
   /// a new spreadsheet with [sheetName] and populates the spreadsheet with
   /// entries in the [insertionOrderList].
-  void populate(List<InsertionOrder> insertionOrderList) async {
+  ///
+  /// If user checks 'highlight underpacing insertion orders' on the UI, then
+  /// [highlightUnderpacing] resolves as true, and false otherwise.
+  void populate(List<InsertionOrder> insertionOrderList,
+      bool highlightUnderpacing) async {
     await Office.onReady(allowInterop((info) async {
       final tableName = 'Performance_Data_Table';
 
       await ExcelJS.run(allowInterop((context) async {
         // Adds a new worksheet.
         final sheet = context.workbook.worksheets.add('Query');
 
+        // Adds and formats underpacing info.
+        sheet.getRange('Query!A1:B2')
+          ..format.font.name = _fontName
+          ..format.font.size = _fontSize
+          ..format.font.bold = true
+          ..format.horizontalAlignment = _centerAlignment
+          ..values = [
+            ['highlight underpacing:', '$highlightUnderpacing'],
+            ['threshold: ', 0.9]
```

Comment:
pull this out into a const

Reply:
Can't turn this into a const since $highlightUnderpacing is a variable, but I will pull this out into a static final.